app/Http/Controllers/DashboardController.php

```diff
@@ -101,27 +104,45 @@ public function calls()
      */
     public function notes()
     {
+        $client = CouchDbHelper::getAccountDatabase(auth()->user()->account->id);
+
         $notesCollection = collect([]);
-        $notes = auth()->user()->account->notes()->favorited()->get();
+        $response = $client->descending(true)->include_docs(true)->getView('notes', 'favorites');
```

Comment:
Could the `$client` come from the Account object instead? So you would call it with `auth()->user()->account->client` or something, instead of having to request the client from the helper every time?

I wonder if we could create a method in the $client object to get the objects we want from the DB, like `$client->get('notes', 'favorites);`... but would we cover all the cases we need... I don't know.

Reply:
1) `auth()->user()->account->client` is a great idea.

2) I'll think about this.